test/python/algorithms/test_adapt_vqe.py

```diff
@@ -0,0 +1,82 @@
+""" Test of the Adaptive VQE ground state calculations """
```

Comment:
```suggestion
""" Test of the AdaptVQE minimum eigensolver """
```
`AdaptVQE` is the official name of this algorithm